src/View.cpp

```diff
@@ -194,9 +204,11 @@ bool CJNIViewInputDevice::isVirtual() const
 
 bool CJNIViewInputDevice::supportsSource(int source) const
 {
-  return call_method<jboolean>(m_object,
-    "supportsSource", "(I)Z",
-    source);
+  if (GetSDKVersion() >= 21)
```

Comment:
21 is our min SDK version, so no need for this check.

Reply:
updated